### ISSUE TITLE
Create a pipeline to bump the stemcell version

### ISSUE THREAD

**Issue by f0rmiga** [SUSE/kubecf#28]
Once a new stemcell is released, the pipeline should bump the stemcell version on `//deploy/helm/scf/values.yaml` and open a pull request against the kubecf repo.

**Comment by svollath** [comment]
Refer to "registry.opensuse.org/cloud/platform/fissile-stemcell/sle_15_sp1/fissile-stemcell-sle15:latest" when we fix the tag for a proper version

**Comment by f0rmiga** [comment]
@svollath Could you be more specific?

**Comment by f0rmiga** [comment]
Unblocked this task since the pipeline work itself is fine to be done. Which stemcell to use can be configured in the future.

**Comment by f0rmiga** [comment]
Closing this in favour of https://github.com/SUSE/kubecf/issues/172.
